src/nodes/fmod_bank_loader.cpp

```diff
@@ -27,14 +29,31 @@ const Array& FmodBankLoader::get_bank_paths() const {
     return bank_paths;
 }
 
+bool FmodBankLoader::_property_can_revert(const StringName& p_property) const {
+    if (p_property == StringName(bank_path_property_name)) {
```

Comment:
Nitpick, but I'd rather have the StringName stored in a FmodStringName singleton, like Godot does internally, instead of hashing/instantiating it every single call (which the Godot editor calls for every property of an object each time the inspector is refreshed).

(The singleton has to be initialized and destroyed alongside the extension lifecycle or Godot will complain about StringName leaks).